packages/terra-brand-footer/src/BrandFooter.jsx

```diff
@@ -104,20 +84,11 @@ const BrandFooter = ({
   ),
   customProps.className);
 
-  let processedSections;
-  if (links.length > 0 && sections.length === 0) {
-    processedSections = [{
-      headerText: '',
-      links,
-      id: 0,
-    }];
-  } else {
-    processedSections = sections;
+  const processedSections = sections;
 
-    // Assign ids to use as keys
-    for (let i = 0; i < processedSections.length; i += 1) {
-      processedSections[i].id = i;
-    }
+  // Assign ids to use as keys
+  for (let i = 0; i < processedSections.length; i += 1) {
+    processedSections[i].id = i;
   }
 
   // The old links prop can't have section headers. Needed for vertical layout to insert padding to keep columns without headers aligned
```

Comment:
Is this comment still applicable?

Reply:
Oops!! missed it. Removed it here https://github.com/cerner/terra-framework/pull/1167/commits/64bf30125e4e5ebb2a7adc5693bbdc2b565fbb51